src/features/courses/course-types.ts

```diff
@@ -299,6 +307,33 @@ export interface GetCalculatedRendererParamsResponse {
     showSolutions: number;
 }
 
+export interface GradeOptions {
+    studentGrade: StudentGrade;
+    topic: CourseTopicContent;
+    question: CourseWWTopicQuestion;
+
+    solutionDate: moment.Moment;
```

Comment:
Why is this date a Moment when `timeOfSubmission` is a Date? I feel like we should keep them consistent.

Reply:
yeah i'm back and forth on this, the reason is that solution date is calculated from moment, so to do this I would have to convert from moment back to date, pass it in and the function would convert it to a moment

The other fields come from the database model and all three are used as moment in this function

Instead of changing this to a date I could change timeofSubmission into a moment I suppose